include/rdma/fabric.h

```diff
@@ -221,11 +221,12 @@ enum {
 };
 
 /* Mode bits */
-#define FI_CONTEXT		(1ULL << 59)
-#define FI_MSG_PREFIX		(1ULL << 58)
-#define FI_ASYNC_IOV		(1ULL << 57)
-#define FI_RX_CQ_DATA		(1ULL << 56)
-#define FI_LOCAL_MR		(1ULL << 55)
+#define FI_CONTEXT			(1ULL << 59)
+#define FI_MSG_PREFIX_DEPRECATED	(1ULL << 58)
+#define FI_ASYNC_IOV			(1ULL << 57)
+#define FI_RX_CQ_DATA			(1ULL << 56)
+#define FI_LOCAL_MR			(1ULL << 55)
+#define FI_MSG_PREFIX			(1ULL << 54)
```

Comment:
The simplest fix would be to keep FI_MSG_PREFIX as is, and only add a new mode bit: FI_MSG_PREFIX2